### PR TITLE
[DONE] Add optionnal proxy URL for request coming from Aristote

### PR DESCRIPTION
**Pull request by gcondess**
Aristote IA as an infrastructure based on Kubernetes in order to automaticaly générate or delete backends servers adaptatively to the current load. 
Those backends servers are directly requesting the POD server : Once to retrieve the URL of the audio file (witch is the base data for the IA treatment) and in second time to notify POD that the treatment is achieved.
So if the POD server not accessible from the all the WWW, the Aristote backend's will not be able to retrieve the audio file for treatment. That can be a problem on POD test servers that are not open to all the web for security reasons.
This code allow to specify a proxy URL (accessible from all the web) in order to catch Aristote backend's requests and redirect them to the POD test server (via the internal institution network)

### CODE REVIEW

File: CONFIGURATION_EN.md
Comment (Badatos) [inline CONFIGURATION_EN.md:1]: il ne faut pas modifier les fichiers CONFIGURATION_XX.md, ils sont générés automatiquement à partir des infos du configurations.json

Comment (gcondess) [reply CONFIGURATION_EN.md:1]: Ok merci pour l'info ( j'avais pas compris ça !

**Comment by LoicBonavent** [comment]
Je pense qu'il faudrait remplacer comming par coming, et de [WIP] en [DONE], non ?

**Review by Badatos — APPROVED** [review]

**Review by LoicBonavent — APPROVED** [review]
Merci Guillaume

**Comment by Badatos** [comment]
Reste la faute de frappe à corriger dans le json et on pourra fusionner ;)

**Pull request merged**